app/components/search_result/location_component.html.erb

```diff
@@ -6,18 +6,15 @@
       <th scope='col' class='col-xs-6'>Status</th>
     </tr>
   </thead>
-  <% if bound_with_child? %>
-    <tbody class="bound-with-note">
+  <% if bound_with? %>
+    <tbody>
       <tr>
-        <th scope='col'>
-          Some records bound together
+        <th colspan='2'>
```

Comment:
I don't think this is semantically a `<th>` anymore. Can it be a `<td>`?  Then then we don't need to style font-weight.

Reply:
yep. done.